src/lib.rs

```diff
@@ -481,16 +487,21 @@ impl LineParser {
 ///
 /// For basic usage, see the crate-level documentation.
 /// Note that once `next` returns an error, the result of further calls is undefined.
-pub struct PropertiesIter<R: Read> {
-  lines: LogicalLines<NaturalLines<R>>,
+pub struct PropertiesIter<'a, R: Read> {
```

Comment:
Depends... would you consider a `Box<Encoding>` in `NaturalLines`? That could also make it possible with no breaking changes.